storage/src/mmr/benches/build_mmr.rs

```diff
@@ -0,0 +1,26 @@
+use commonware_cryptography::{Digest, Sha256};
+use commonware_storage::mmr::mem::Mmr;
+use criterion::{criterion_group, Criterion};
+
+fn bench_build_mmr(c: &mut Criterion) {
+    let element = Digest::from_static(&[100u8; 32]);
+    c.bench_function("build_mmr", |b| {
+        let mut mmr = Mmr::<Sha256>::new();
+        // bootstrap w/ 5M elements
+        for _ in 0..5_000_000 {
+            mmr.add(&element);
+        }
+        // time adding 1M more
+        b.iter(|| {
```

Comment:
In other tests, I use `iter_batched` to perform setup: https://bheisler.github.io/criterion.rs/criterion/struct.Bencher.html#method.iter_batched

Even though it may look like you aren't affecting the elapsed time with this approach, you are: https://bheisler.github.io/criterion.rs/criterion/struct.Bencher.html#method.iter

Reply:
If you generate the values outside of `bench_function`, you can use `b.iter`